tensorflow-frameworks/tensorflow-utils/pom.xml

```diff
@@ -0,0 +1,62 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.tensorflow</groupId>
+        <artifactId>tensorflow-frameworks</artifactId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>tensorflow-utils</artifactId>
```

Comment:
You can move classes in this artifact directly in `tensorflow-tools`, its main purpose is to provide such generic utilities.

But looking again at your PR, it seems that those classes (`Pair` and `Tuple2`) are not being used anywhere, so I suggest we simple drop them.

Reply:
Agreed

Reply:
I removed `tensorflow-utils`